axon-framework/monitoring/tracing.md

```diff
@@ -0,0 +1,535 @@
+| `${SnapshotterClass}.createSnapshot($aggregateClass)`                       | A snapshot creation task is being submitted. Depending on performance, the executor might take a while to pick it up. |
+| `${SnapshotterClass}.createSnapshot($aggregateClass, $aggregateIdentifier)` | The `Snapshotter` is now creating the snapshot.                                                                       |
+
+Note that the root trace does not contain the aggregate identifier,
+so any snapshot invocation of the same aggregate type is grouped together by the APM tool.
+
+### Sagas
+Sagas are a special type of event processor that can invoke multiple saga's for a single event. 
+Because of this the `AbstractSagaManager` has been instructed with additional tracing information.
+These spans are descendants of an event processor span that invokes the manager.
+
+| Trace name                                                                  | Description                                          |
+|-----------------------------------------------------------------------------|------------------------------------------------------|
+| `SagaManager[${SagaTypeName}].invokeSaga ${sagaIdentifier}`                 | A matching saga has been found and is being invoked. |
+| `SagaManager[${SagaTypeName}].startNewSaga`                                 | The manager is constructing a new saga.              |
+
+
+### Queries
+Queries support tracing in all of their forms. In order to be clear about how they work, 
+this section is split based upon the query's type. 
+For all types, the created spans will differ based on whether Axon Server is used or not. 
+The spans that are only available with Axon Server are marked as such.
+
+#### Direct queries
+Direct queries fetch a single result (either a single item or a single list) and receive no updates. 
+Their traces will look like the following table:
+
+| Trace name                                                | Description                                                                           |
+|-----------------------------------------------------------|---------------------------------------------------------------------------------------|
+| `AxonServerQueryBus.query(${queryName})`                  | (Only with Axon Server) The requesting service is dispatching the query.              |
+| `QueryProcessingTask(${queryName})`                       | (Only with Axon Server) The handling service is handling the query request in a task. |
+| `SimpleQueryBus.query(${queryName})`                      | The handling service is handling the query.                                           |
+| `AxonServerQueryBus.ResponseProcessingTask(${queryName})` | (Only with Axon Server) The requesting service is processing the response.            |
+
+#### Streaming queries
+Streaming queries, introduced in Axon Framework 4.6.0, look very similar to the traces of a Direct query. 
```

Comment:
The selected reference guide version will automatically state it's Axon 4.6.0.
Hence, there's no need to specify that again.

```suggestion

Streaming queries look very similar to the traces of a Direct query. 
```